BibBuddy/app/src/main/java/de/bibbuddy/LibraryRecyclerViewAdapter.java

```diff
@@ -111,6 +119,24 @@ public void setLibraryList(List<ShelfItem> libraryList) {
     this.libraryList = libraryList;
   }
 
+  /**
+   * This method fetches the number of items selected in the recyclerView.
+   *
+   * @return returns the selected recyclerView items.
+   */
+  public List<LibraryItem> getSelectedLibraryItems() {
+    List<LibraryItem> selectedItems = new ArrayList<>();
+    if (parent != null) {
+      int itemNumber = parent.getChildCount();
+      for (int i = 0; i < itemNumber; i++) {
+        if (parent.getChildAt(i).isSelected()) {
+          selectedItems.add(libraryList.get(i));
+        }
+      }
+    }
+    return selectedItems;
```

Comment:
Same suggestions as in the other adapters.
```suggestion
    List<LibraryItem> selectedItems = new ArrayList<>();
    
    if (parent != null) {
      for (int i = 0; i < parent.getChildCount(); i++) {
        if (parent.getChildAt(i).isSelected()) {
          selectedItems.add(libraryList.get(i));
        }
      }
    }
    
    return selectedItems;
```

Reply:
Thanks for the suggestion, done!